blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/analytics/AnalyticsConfigService.java

```diff
@@ -0,0 +1,62 @@
+package io.jenkins.blueocean.service.embedded.analytics;
+
+import com.google.common.base.Supplier;
+import com.google.common.base.Suppliers;
+import hudson.ProxyConfiguration;
+import io.jenkins.blueocean.commons.JsonConverter;
+import jenkins.model.Jenkins;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.Proxy;
+import java.net.URL;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public final class AnalyticsConfigService {
+
+    private static final Logger LOGGER = Logger.getLogger(AnalyticsConfigService.class.getName());
+
+    private static final String ANALYTICS_CONFIG_URL = "https://jenkins.io/projects/blueocean/analytics.json";
+
+    public static AnalyticsConfig get() {
+        return configSupplier.get();
+    }
+
+    private static Supplier<AnalyticsConfig> configSupplier = Suppliers.memoizeWithExpiration(() -> {
+        AnalyticsConfig config;
```

Comment:
Would I be correct in assuming that this method is going to be executed only once per instance, on boot?

Reply:
Its memoized so that the stored value is refreshed every day